crates/modelardb_common/src/parser.rs

```diff
@@ -742,11 +737,28 @@ pub fn parse_sql_expression(df_schema: &DFSchema, sql_expr: &str) -> Result<DFEx
 /// which was released under version 2.0 of the Apache License.
 ///
 /// [rewrite_expr.rs]: https://github.com/apache/arrow-datafusion/blob/main/datafusion-examples/examples/rewrite_expr.rs
-struct EmptyContextProvider {
+struct ParserContextProvider {
     options: ConfigOptions,
+    udfs: HashMap<String, Arc<ScalarUDF>>,
```

Comment:
Maybe add doc comments explaining the individual struct fields.

Reply:
Fixed.